### ISSUE TITLE
problem with isFloat()

### ISSUE THREAD

**Issue by mbyervand** [dart-league/validators#14]
isFloat() recognizes this length of integer  "66666666666" as a float .

**Comment by mbyervand** [comment]
Was this problem solved?

**Comment by luisvt** [comment]
no, sorry, I haven't had time for this. Feel free to create a PR with the needed changes.
